torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9094,7 +9094,9 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            skips=(
                # cumprod does not handle correctly out= dtypes
                DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_out'),
-               DecorateInfo(unittest.expectedFailure, 'TestCompositeCompliance', 'test_backward'),
+               # RuntimeError: "prod_cpu" not implemented for 'BFloat16'
+               DecorateInfo(unittest.expectedFailure, 'TestDecomp', 'test_comprehensive',
+                            dtypes=(torch.bfloat16,), device_type='cpu'),
```

Comment:
Regression: The new path (with Python Dispatch) uses prod which doest not support BFloat16 on CPU.